src/mlfmu/fmu_build/templates/onnx_fmu/onnxFmu.cpp

```diff
@@ -41,7 +45,12 @@ std::wstring OnnxFmu::formatOnnxPath(cppfmu::FMIString fmuResourceLocation)
     if (startPath == L"file:///") {
         path = endPath;
     }
-    return path;
+    // save to onnxPath_ (wstring for Windows, else string)
+#ifdef _WIN32
+    onnxPath_ = path;
+#else
+    onnxPath_ = std::string(path.begin(), path.end());
+#endif
```

Comment:
This looks like a good solution! 
Luckily it seemed the simple solution i suggested worked